cns/service/main.go

```diff
@@ -789,6 +789,13 @@ func main() {
 		if platform.HasMellanoxAdapter() {
 			go platform.MonitorAndSetMellanoxRegKeyPriorityVLANTag(rootCtx, cnsconfig.MellanoxMonitorIntervalSecs)
 		}
+		// if swiftv2 scenario is enabled, we need to initialize the Service Fabric (standalone) swiftv2 middleware to process IP configs requests
+		if cnsconfig.EnableSwiftV2 {
+			if cnsconfig.SWIFTV2Mode == configuration.SFSWIFTV2 {
```

Comment:
we can get rid of this config IMO, since each SWIFT v2 scenario is operated in 2 different channel mode without conflict.

Reply:
ack removed this config altogether, could be added if required later